### ISSUE TITLE
[jss-cli] jss to css conversion

### ISSUE THREAD

**Issue by nwmcsween** [jsstyles/jss#187]
trying to run jss on any file results in:
```
/home/nwmcsween/.nvm/versions/node/v5.5.0/lib/node_modules/jss-cli/bin/cli.js:33
    var jss = converterExtMap[ext](code)
                                  ^

TypeError: converterExtMap[ext] is not a function
```

**Comment by watilde** [comment]
What kind of file extension did the target file have? Current code seems only can handle `.css` file.
> https://github.com/jsstyles/cli/blob/master/bin/cli.js#L27

**Comment by kof** [comment]
It expects that you convert .css files. To enable different extensions, we need an option which says that  the source file is of type ... and extension should be ignored.

**Comment by kof** [comment]
@watilde it outputs to the stdout ...

**Comment by nwmcsween** [comment]
I assumed the jss command line program converted jss to css or css to jss, it seems there is no via cmdline to simply spit out css from a js file? I dislike the idea of having to create a 'runner' for each project to build the css.

**Comment by kof** [comment]
The prob with jss-> css conversion is that it depends on your jss setup, which means we can't just require the file with files and convert it to css, wee need to require your jss setup too. I have no idea how to do that right.

**Comment by nwmcsween** [comment]
Are you talking about if a file uses plugins? Why not have it as switches to the jss cmd?

**Comment by kof** [comment]
if you jss style relies on plugins. For e.g. using extend or default unit etc. or even a custom user plugin. You gonna need the users jss setup in order to convert it from jss...

**Comment by kof** [comment]
Of course you could do something like jss ./style.js --plugins extend, default-unit ... user-plugin ...  etc ....

**Comment by kof** [comment]
I am not sure if its worth something because to write down this cli call will take more than to  write a js file, require your jss setup and then do console.log(sheet.toString())

**Comment by kof** [comment]
Another option would be to pass the jss setup: `jss ./style.js --jss ./jss-setup.js`

**Comment by kof** [comment]
If no --jss option is defined, it will create a plain instance without plugins.

**Comment by kof** [comment]
Would you like to send me a pr? @nwmcsween @watilde

**Comment by kof** [comment]
Here is the final spec:

- Create "jss" option, when present its a path to a file which exports a jss instance and will be required, it is optional, default is require('jss')
- Create "plugins" option. Its optional, when passed it can be one or more plugin names (comma separated) as they are specified in npm and installed locally. If specified, every pluginn will be required and jss.use(plugin()) will be called.
- Converter needs to know "js" and "jss" file extensionsa and decide based on that in which direction to convert.

**Comment by kof** [comment]
Ok doing it now.

**Comment by kof** [comment]
released version 3.0 with jss2css converter support, please try it out and let me know if it doesn't work for you)
